subsys/net/lib/lwm2m_client_utils/location/location_assistance.c

```diff
@@ -52,58 +53,188 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #define GNSS_ASSIST_ELEVATION_MASK			8
 
 static struct lwm2m_ctx *req_client_ctx;
-
+#ifdef CONFIG_ZTEST
+/* faster timeout for test purspose */
```

Comment:
Why we need faster timeout for testing?
Don't we run the tests on native_posix where we don't even use the real-time scheduler?

Reply:
Let me check how fast it run.